### PR TITLE
Dev branch1

### PR DESCRIPTION
**Pull request by num39**

### CODE REVIEW

**Comment by num39** [comment]
please review this code

File: cst438_gradebook/src/main/java/com/cst438/controllers/GradeBookController.java [hunk 1]
```diff
@@ -47,8 +49,7 @@ public class GradeBookController {
 	@GetMapping("/gradebook")
```
Comment (jmann-csumb) [inline cst438_gradebook/src/main/java/com/cst438/controllers/GradeBookController.java:49]: Viewing gradbook code looks good, grades can be searched by email for student class assignment.

**Review by jmann-csumb — COMMENTED** [review]
Gradebook controller code looks good and it is dvided into multiple sections as view grades, crate new assignment, delete assignment by ID number and update assignment by ID number. It seems to meet the requirement.

File: cst438_gradebook/src/main/java/com/cst438/controllers/GradeBookController.java
Comment (jmann-csumb) [inline cst438_gradebook/src/main/java/com/cst438/controllers/GradeBookController.java:1]: Looks good, but sql setting should not be shared

File: cst438_gradebook/src/main/java/com/cst438/domain/AssignmentListDTO.java
Comment (jmann-csumb) [inline cst438_gradebook/src/main/java/com/cst438/domain/AssignmentListDTO.java:1]: Jason DTO parameters/ variables are clearly defined along with public class.

**Pull request merged**